content-src/components/Base/_Base.scss

```diff
@@ -54,6 +54,11 @@ main {
   height: 100vh;
 }
 
+
+.as-err.ds-error-fallback.ds-error-fallback {
```

Comment:
Probably want just `.ds-error-fallback` it renders as `class="ds-error-fallback as-error-fallback"`

Reply:
Unfortunately due the way the css files are ordered the specificity isn't high enough without this, I could alternatively move this class to the error css file

Reply:
oh sorry, this is supposed to be `.as-error-fallback.ds-error-fallback`